doc-content/enterprise-only/weblogic/wls-configure-embedded-engine-proc.adoc

```diff
@@ -0,0 +1,260 @@
+ifdef::PAM[]
+https://access.redhat.com/solutions/3405361[What is the mapping between Red Hat Process Automation Manager and the Maven library version?].
+endif::[]
+ifdef::DM[]
+https://access.redhat.com/solutions/3363991[What is the mapping between Red Hat Decision Manager and the Maven library version?].
+endif::[]
+
+.Procedure
+
+. Declare the {PRODUCT_BA} BOM in the `pom.xml` file:
++
+====
+[source,xml,subs="attributes+"]
+----
+<dependencyManagement>
+ <dependencies>
+  <dependency>
+   <groupId>com.redhat.ba</groupId>
+   <artifactId>ba-platform-bom</artifactId>
+   <version>{BOM_VERSION}</version>
+   <type>pom</type>
+   <scope>import</scope>
+  </dependency>
+ </dependencies>
+</dependencyManagement>
+<dependencies>
+<!-- Your dependencies -->
+</dependencies>
+----
+====
++
+. Declare dependencies required for your project in the `<dependencies>` tag. After you import the product BOM into your project, the versions of the user-facing product dependencies are defined so you do not need to specify the `<version>` sub-element of these `<dependency>` elements. However, you must use the `<dependency>` element to declare dependencies which you want to use in your project.
++
+* For a basic {PRODUCT} project, declare the following dependencies, depending on the features that you want to use:
++
+ifdef::PAM[]
+[id='_embedded_jbpm_engine_dependencies']
+.Embedded {PROCESS_ENGINE} dependencies
+[source,xml,subs="attributes+"]
+----
+<!-- Public KIE API -->
+<dependency>
+  <groupId>org.kie</groupId>
+  <artifactId>kie-api</artifactId>
+</dependency>
+
+<!-- Core dependencies for {PROCESS_ENGINE} -->
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-flow</artifactId>
+</dependency>
+
+<dependency>
+  <artifactId>jbpm-flow-builder</artifactId>
+</dependency>
+
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-bpmn2</artifactId>
+</dependency>
+
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-runtime-manager</artifactId>
+</dependency>
+
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-persistence-jpa</artifactId>
+</dependency>
+
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-query-jpa</artifactId>
+</dependency>
+
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-audit</artifactId>
+</dependency>
+
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-kie-services</artifactId>
+</dependency>
+
+<!-- Dependency needed for default WorkItemHandler implementations. -->
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-workitems-core</artifactId>
+</dependency>
+
+<!-- Logging dependency. You can use any logging framework compatible with slf4j. -->
+<dependency>
+  <groupId>ch.qos.logback</groupId>
+  <artifactId>logback-classic</artifactId>
+  <version>${logback.version}</version>
+</dependency>
+----
+
+* For {WEBLOGIC}, user needs to declare the following hibernate dependencies in the `pom.xml` file:
```

Comment:
needs to  - do not use, use must
so
, you must declare

Reply:
I would reword and remove the note at the bottom:
If you intend to use {PROCESS_ENGINE} in a {WEBLOGIC} {PRODUCT} application,  declare the following hibernate dependencies in the `pom.xml` file:
Note, that what I have written might not be correct so check with your SME to fine tune, but the point is put the condition in the intro statement rather than a note below.